pwa/get-core-docs.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+git clone https://github.com/api-platform/core /api-platform/core
+
+versions=("3.1" "main")
+for version in "${versions[@]}"
+do
+	cd /api-platform/core
+	git checkout $version
+	composer install --no-dev
+	composer require webonyx/graphql-php
```

Comment:
These optimizations broke the script because we need all dependencies to generate the documentation. When testing with `--no-update` flag and `-a`, an error occurred because of missing dependencies

Reply:
yeah the order of things are wanted to generate a full reference...